Cargo.toml

```diff
@@ -15,5 +15,5 @@ members = [
     "agents/processor",
     "tools/kms-cli",
     "tools/nomad-cli",
-    # "tools/balance-exporter"
+    "tools/balance-exporter",
```

Comment:
Hmm I think this can stay commented out. We're gonna delete `balance-exporter` anyways: https://github.com/nomad-xyz/rust/issues/38

Reply:
shrug. if it's in the repo we should build it 😅  let's remove it when it's gone